modules/heat_conduction/test/tests/gap_heat_transfer_mortar/tests

```diff
@@ -284,4 +284,17 @@
                   'HEX20 elements and match reasonably well with the node-to-segment approach with a '
                   'refined mesh.'
   []
+
+  [closed_gap_prescribed_pressure]
+    design = 'GapFluxModelPressureDependentConduction.md'
+    issues = '#20658'
+    input = 'closed_gap_prescribed_pressure.i'
+    type = CSVDiff
+    csvdiff = 'closed_gap_prescribed_pressure_out.csv'
+    ad_indexing_type = 'global'
+    cli_args = "Mesh/allow_renumbering=false"
+    requirement = 'We shall be able to compute the thermal contact across a closed gap as a function '
+                  'of contact pressure, and the material thermal conductivities and hardness values '
```

Comment:
This is technically testing an LM that could very well be the contact pressure, but it's not really. One suggestion would be to add another test using the contact mechanics action that makes your object pull the actual variable LM describing the interpolated normal pressure on the lower dimensional domain.